components/Spaces/Chat/index.js

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import { IconButton, TextField } from "@material-ui/core"
+import Paper from "@material-ui/core/Paper"
+import SendIcon from '@material-ui/icons/Send';
+
+import Conversation from './Conversation'
+import { useConversation } from '../../../context/ConversationProvider'
+
+const Chat = () => {
+  const [text, setText] = useState('')
+  const [error, setError] = useState(false)
+  const { sendMessage } = useConversation()
+
+  const submitHandler = (e) => {
+    e.preventDefault()
+
+    const messageDidSend = sendMessage(text)
+    setError(!messageDidSend)
+    setText('')
+    setTimeout(() => {
+      setError(false)
+    }, 2000)
+  }
+
+  const keyPressHandler = (e) => {
+    if (e.keyCode == 13) {
+      submitHandler(e)
+    }
+  }
+
+  return (
+    <>
+      <Paper className="flex flex-col h-96 min-h-full w-9/12 max-w-lg" elevation={3}>
```

Comment:
waitt I didn't even know you could do that, that's pretty cool

Reply:
aha thanks